.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish site
+
+on:
+  workflow_call:
+    inputs:
+      destination_dir:
+        required: true
+        type: string
+
+jobs:
+  publish-site:
```

Comment:
Should we name this something else if we want to deploy snaps-simulator from here too?

Reply:
IIRC you can't pass down environments in input variables

Reply:
I've renamed this workflow to `publish-environment.yml`.